include/ofi_shm.h

```diff
@@ -156,6 +156,7 @@ struct smr_cmd {
 
 #define SMR_INJECT_SIZE		4096
 #define SMR_COMP_INJECT_SIZE	(SMR_INJECT_SIZE / 2)
+#define SMR_SAR_SIZE		(1 << 14)	//16k
```

Comment:
It's better to unify here, either use 16384 or change SMR_INJECT_SIZE to (1 << 12)